pkg/dogstatsd/server.go

```diff
@@ -833,3 +833,8 @@ func FormatDebugStats(stats []byte) (string, error) {
 
 	return buf.String(), nil
 }
+
+// SetExtraTags sets extra tags to the server
```

Comment:
I would rephrase this comment -- what are the extra tags set on exactly? I wouldn't say they are set "to the server"